lib/librato/rack/tracker.rb

```diff
@@ -16,6 +14,7 @@ class Tracker
       def initialize(config)
         @config = config
         collector.prefix = config.prefix
+        collector.aggregate.add_tags(qualified_tags)
```

Comment:
I can see why we need to modify the tags after instantiation in this case, but maybe this should be `set_tags` or something instead - are we effectively replacing here?